backend/src/app.ts

```diff
@@ -16,6 +16,8 @@ class App {
     this.associationSetup.sync();
   }
   private config(): void {
+    const cors = require('cors')
+    this.app.use(cors())
```

Comment:
thanks for doing this!

Reply:
@JDziewonski98 this is a fairly important backend change - is it good by you?

Reply:
will have to change this come deployment but its not a big deal right now